src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c

```diff
@@ -201,41 +184,41 @@ void EnPoSisters_Init(Actor* thisx, PlayState* play) {
     ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawCircle, 50.0f);
     SkelAnime_Init(play, &this->skelAnime, &gPoeSistersSkel, &gPoeSistersSwayAnim, this->jointTable, this->morphTable,
                    12);
-    this->unk_22E.r = 255;
-    this->unk_22E.g = 255;
-    this->unk_22E.b = 210;
-    this->unk_22E.a = 255;
+    this->sisterColor.r = 255;
+    this->sisterColor.g = 255;
+    this->sisterColor.b = 210;
+    this->sisterColor.a = 255;
     this->lightNode = LightContext_InsertLight(play, &play->lightCtx, &this->lightInfo);
     Lights_PointGlowSetInfo(&this->lightInfo, this->actor.home.pos.x, this->actor.home.pos.y, this->actor.home.pos.z, 0,
                             0, 0, 0);
     Collider_InitCylinder(play, &this->collider);
     Collider_SetCylinder(play, &this->collider, &this->actor, &sCylinderInit);
     CollisionCheck_SetInfo(&this->actor.colChkInfo, &sDamageTable, &sColChkInfoInit);
-    this->unk_194 = PARAMS_GET_U(thisx->params, 8, 2);
-    this->actor.naviEnemyId = this->unk_194 + NAVI_ENEMY_POE_SISTER_MEG;
+    this->sisterID = PARAMS_GET_U(thisx->params, 8, 2);
```

Comment:
Looks like these generic `PARAMS_GET_` macros can be given specific names

`#define POE_SISTER_GET_ID(thisx) PARAMS_GET_U(thisx->params, 8, 2)`
and similar, to go in the header